src/core/installAddon.js

```diff
@@ -165,9 +180,26 @@ export function makeMapDispatchToProps({ WrappedComponent, src }) {
 
 const userAgentOSToPlatform = {
   [USER_AGENT_OS_ANDROID]: OS_ANDROID,
-  [USER_AGENT_OS_LINUX]: OS_LINUX,
   [USER_AGENT_OS_MAC]: OS_MAC,
   [USER_AGENT_OS_WINDOWS]: OS_WINDOWS,
+  // Not all of these are strictly Linux but giving them a Linux XPI
```

Comment:
Should we maybe file a bug to rename "Linux" to "*NIX" or something in the UI? I dunno, both options are kinda meh but I wonder which is better...